src/qri/userProfile.ts

```diff
@@ -15,37 +15,53 @@ export interface UserProfile {
   photo: string
   poster: string
   twitter: string
-  PeerIDs: string[]
-  NetworkAddrs: string[]
+  peerIDs: string[]
+  networkAddrs: string[]
   id: string
   currentKey: string
-  EmailConfirmed: boolean
+  emailConfirmed: boolean
   isAdmin: boolean
 }
 
-export const NewUserProfile = () => {
+export const NewUserProfile = (d: Record<string, any>): UserProfile => {
```

Comment:
We have a similar `NewDataset` function that I've found useful. Converted this to be similar.